app/views/chatrooms/_member.html.erb

```diff
@@ -0,0 +1,17 @@
+<% @member.each do |member| %>

```

Comment:
em có làm render partial with collection mà nó toàn báo missing template mặc dù em đã tạo file _member và render @member rồi ạ

Reply:
tìm hiểu kỹ chưa em ei??

Reply:
https://viblo.asia/p/render-partials-in-rails-ZjleadVEGqJ
có một thuyết âm mưu tăng view không hề nhẹ ở đây